### ISSUE TITLE
Provide a way of configuring the location of email templates

### ISSUE THREAD

**Issue by samtgarson** [zenorocha/react-email#248]
I'm experimenting with adding this package to an already existing large monorepo, and I don't want to keep the email templates at the top level but it doesn't seem like there's a way of configuring this (it's hard coded [here](https://github.com/zenorocha/react-email/blob/main/packages/react-email/source/utils/contants.ts#L5)).

The `dev` command doesn't accept any options at the moment, is this something you're already considering?

(P.S. that constants file has a typo in the filename)

**Comment by bukinoshita** [comment]
Yes, we can the default directory to be `emails`, but this should be optional and the user can change it they want it.

**Comment by emroot** [comment]
@bukinoshita any update here? Kind of a blocker at the moment

**Comment by bukinoshita** [comment]
> @bukinoshita any update here? Kind of a blocker at the moment

Not yet, i'll take a look on Sunday or next week.

**Comment by m-shaka** [comment]
Can I take this? Our team wants this right now!

**Comment by bukinoshita** [comment]
Feel free to take @m-shaka! Thanks

**Comment by zenorocha** [comment]
@samtgarson @emroot @co-sic @GRA0007 just published `react-email@1.7.6` which contains that option.

Can you try now?

https://github.com/resendlabs/react-email/pull/384
https://github.com/resendlabs/react-email/pull/401

**Comment by m-shaka** [comment]
@zenorocha I found `react-email@1.7.6` seems to refer to an outdated script, while I'm not sure why it's there.
`bin.email` in pakcage.json is `./dist/index.js`, but actually new generated file is `dist/source/index.js`.
https://github.com/resendlabs/react-email/blob/main/packages/react-email/package.json#L6

You can explore here
https://www.npmjs.com/package/react-email?activeTab=explore
![image](https://user-images.githubusercontent.com/12842076/215239589-d79c5d7e-f1a6-411b-b8a0-920132568a00.png)
![image](https://user-images.githubusercontent.com/12842076/215239605-ff974dc4-f8e2-4797-802e-50c8f116ebd3.png)

**Comment by zenorocha** [comment]
@m-shaka just built a new version, `1.7.7`, can you try now?
